staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -1216,6 +1216,13 @@ message Container {
     // MUST be identical to that of the corresponding ContainerConfig used to
     // instantiate this Container.
     map<string, string> annotations = 9;
+    // Reference to the unique identifier of the image. The image_ref above has
```

Comment:
suggest the "For most runtimes.." sentence above in the description of image_ref

and for image_id suggest:

     // Reference to the unique identifier of the image, on the node, as returned in the image service apis.

Reply:
Changed as suggested.